src/App.tsx

```diff
@@ -1,24 +1,202 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import classNames from 'classnames';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo, Errors, Filter } from './types/Todo';
+import { Form } from './components/Form';
+import { TodoList } from './components/Todolist';
+import { Footer } from './components/Footer';
+import { getTodos, postTodo, removeTodo } from './api/todos';
+import { ErrorNotification } from './components/ErrorNotification';
 
-const USER_ID = 0;
+const USER_ID = 6910;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filter, setFilter] = useState<string>(Filter.ALL);
```

Comment:
Instead of using a string for the `filter` state, consider using the `Filter` type from the 'types/Todo'.